mainwindow.ui

```diff
@@ -52,7 +55,7 @@
             <string>Launch</string>
            </property>
            <property name="icon">
-            <iconset resource="qtresources.qrc">
+            <iconset theme="system-run" resource="qtresources.qrc">
```

Comment:
i actually think that the style of the second home icon matches the “Nb” icon here more closely.

but in the end this comes down to two things:

1. whom to give control: designer or user. even GNOME agrees that the answer is “user” when it comes to icon sets.
2. correct choice of [semantic icons](https://developer.gnome.org/icon-naming-spec/). maybe there’s something better than system-run

if you want a “run” icon to be a green right arrow, feel free to use an icon theme that uses it in place of the gear (i see this: ![](https://rawgit.com/NitruxSA/plasma-next-icons/master/Breeze/actions/toolbar/system-run.svg) or this: ![](https://rawgit.com/NitruxSA/plasma-next-icons/22e822d4c4863543377a7819a800b1cf7f8746d0/Breeze/apps/preferences/system-run.svg) when using system-run)